source/v4/system.yaml

```diff
@@ -14,3 +14,21 @@
             $ref: "#/definitions/StatusOK"
         '400':
           $ref: '#/responses/BadRequest'
+  '/config/reload':
+    post:
+      tags:
+        - system
+      summary: Reload the Server configuration
+      description: |
+        Reload the server configuration
```

Comment:
Can the summary be `Reload configuration` and the description be `Reload the configuration file to pick up on any changes made to it.`